omnibus/files/private-chef-cookbooks/private-chef/recipes/oc_bifrost.rb

```diff
@@ -60,4 +60,18 @@
   to oc_bifrost_config
 end
 
+vmargs_config = File.join(oc_bifrost_dir, "vm.args")
+
+template vmargs_config do
+  source "oc_bifrost.vm.args.erb"
+  owner OmnibusHelper.new(node).ownership['owner']
+  group OmnibusHelper.new(node).ownership['group']
+  mode "644"
+  notifies :restart, 'runit_service[oc_bifrost]' unless backend_secondary?
```

Comment:
Note to other reviewers: The differences in the restart strategies across the various apps is matching the already existing differences from the sys.config files for these apps.  My guess is that this can be sorted out so they are all the same, but I don't think it should be a blocker for this.